src/Modules.js

```diff
@@ -2,49 +2,19 @@
 * Core modules and their info files
 */
 module.exports = {
-  'channel': [
-    require('./modules/Channel/Module'), require('./modules/Channel/info')
-  ],
-  'brightness': [
-    require('./modules/Brightness/Module'), require('./modules/Brightness/info')
-  ],
-  'edge-detect': [
-    require('./modules/EdgeDetect/Module'), require('./modules/EdgeDetect/info')
-  ],
-  'ndvi': [
-    require('./modules/Ndvi/Module'), require('./modules/Ndvi/info')
-  ],
-  'invert': [
-    require('./modules/Invert/Module'), require('./modules/Invert/info')
-  ],
-  'crop': [
-    require('./modules/Crop/Module'), require('./modules/Crop/info')
-  ],
-  'colormap': [
-    require('./modules/Colormap/Module'), require('./modules/Colormap/info')
-  ],
-  'decode-qr': [
-    require('./modules/DecodeQr/Module'), require('./modules/DecodeQr/info')
-  ],
-  'fisheye-gl': [
-    require('./modules/FisheyeGl/Module'), require('./modules/FisheyeGl/info')
-  ],
-  'dynamic': [
-    require('./modules/Dynamic/Module'), require('./modules/Dynamic/info')
-  ],
-  'blur': [
-    require('./modules/Blur/Module'), require('./modules/Blur/info')
-  ],
-  'saturation': [
-    require('./modules/Saturation/Module'), require('./modules/Saturation/info')
-  ],
-  'average': [
-    require('./modules/Average/Module'), require('./modules/Average/info')
-  ],
-  'blend': [
-    require('./modules/Blend/Module'), require('./modules/Blend/info')
-  ],
-  'import-image': [
-    require('./modules/ImportImage/Module'), require('./modules/ImportImage/info')
-  ]
+  'channel': require('./modules/Channel'),
```

Comment:
I do really like this simpler way to do this. But it's not 100% clear to me if the added complexity of a module definition is worth it... what do you think?

Reply:
Actually @jywarren the thing is that for adding modules through npm we have to make everything into one file which will be required when we require the module, so this is the only way and I think this introduces good modularity into our system and gives the module freedom to define the info inside js or as json

Reply:
What about loading each of these using `importNewModule()`? Just curious...